pyproject.toml

```diff
@@ -34,7 +34,7 @@ qm-qua = { version = "==1.1.1", optional = true }
 qualang-tools = { version = "==0.14.0", optional = true}
 setuptools = { version = ">67.0.0", optional = true }
 laboneq = { version ="==2.7.0", optional = true }
-qibosoq = { version = "^0.0.3", optional = true }
+qibosoq = { git = "https://github.com/qiboteam/qibosoq.git", branch = "main", optional = true }
```

Comment:
Shall we update this with ">=0.0.4,<0.2"?

Reply:
Yes, I'll wait a moment just for the pypy release to become available